Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595 100%||[Finetuning] Epoch: 1 Avg Loss: 0.3141[Validation] Avg Loss: 0.3187, Avg Dice Score: 0.8986 100%||[Finetuning] Epoch: 2 Avg Loss: 0.3055[Validation] Avg Loss: 0.3176, Avg Dice Score: 0.8978 100% ||[Finetuning] Epoch: 3 Avg Loss: 0.2997[Validation] Avg Loss: 0.3179, Avg Dice Score: 0.8974100%[Finetuning] Epoch: 4 Avg Loss: 0.2974[Validation] Avg Loss: 0.3159, Avg Dice Score: 0.8991 100%||[Finetuning] Epoch: 5 Avg Loss: 0.2943[Validation] Avg Loss: 0.3183, Avg Dice Score: 0.8945 100% |[Finetuning] Epoch: 6 Avg Loss: 0.2927[Validation] Avg Loss: 0.3163, Avg Dice Score: 0.8989 100% ||[Finetuning] Epoch: 7 Avg Loss: 0.2926[Validation] Avg Loss: 0.3223, Avg Dice Score: 0.8915100%[Finetuning] Epoch: 8 Avg Loss: 0.2903[Validation] Avg Loss: 0.3269, Avg Dice Score: 0.8785 100%||[Finetuning] Epoch: 9 Avg Loss: 0.2894[Validation] Avg Loss: 0.3187, Avg Dice Score: 0.8914 100%||[Finetuning] Epoch: 10 Avg Loss: 0.2857143/143 [00:14<00:00, 10.14it/s]143/143 [00:14<00:00, 10.15it/s]143/143 [00:13<00:00, 10.31it/s]143/143 [00:14<00:00, 10.19it/s]143/143 [00:14<00:00, 10.10it/s]143/143 [00:14<00:00, 10.00it/s]143/143 [00:14<00:00, 9.85it/s]143/143 [00:14<00:00, 10.15it/s]143/143 [00:14<00:00, 10.06it/s]143/143 [00:14<00:00, 10.12it/s][Validation] Avg Loss: 0.3156, Avg Dice Score: 0.8976 /home/mikhelson/MedImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to `self.log()` but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'TrainerTest Results Average Loss: 0.5765, Average Dice Score: 0.7327